src/patternfly/components/Menu/examples/Menu.md

```diff
@@ -0,0 +1,837 @@
+              {{/menu-item-text}}
+              {{> menu-item-external-icon}}
+            {{/menu-item-main}}
+          {{/menu-item}}
+          {{> menu-item-action menu-item-action--IsFavorite="true"}}
+        {{/menu-list-item}}
+      {{/menu-list}}
+    {{/menu-group}}
+  {{/menu-body}}
+{{/menu}}
+```
+
+### Option-single-select
+```hbs
+{{#> menu}}
+  {{#> menu-body}}
+    {{#> menu-list}}
+      {{#> menu-list-item}}
+        {{#> menu-item}}
+          {{#> menu-item-main}}
+            {{#> menu-item-text}}
+              Option 1
+            {{/menu-item-text}}
+          {{/menu-item-main}}
+        {{/menu-item}}
+      {{/menu-list-item}}
+      {{#> menu-list-item}}
+        {{#> menu-item}}
+          {{#> menu-item-main}}
+            {{#> menu-item-text}}
+              Option 2
+            {{/menu-item-text}}
+          {{/menu-item-main}}
+        {{/menu-item}}
+      {{/menu-list-item}}
+      {{#> menu-list-item}}
+        {{#> menu-item menu-item--modifier="pf-m-selected"}}
+          {{#> menu-item-main}}
+            {{#> menu-item-icon}}
+              <i class="fas fa-fw fa-table" aria-hidden="true"></i>
+            {{/menu-item-icon}}
+            {{#> menu-item-text}}
+              Option 3
+            {{/menu-item-text}}
+            {{> menu-item-select-icon}}
+          {{/menu-item-main}}
+          {{#> menu-item-description}}
+            Description
+          {{/menu-item-description}}
+        {{/menu-item}}
+      {{/menu-list-item}}
+    {{/menu-list}}
+  {{/menu-body}}
+{{/menu}}
+```
+
+### Option-multi-select
+```hbs
+{{#> menu}}
+  {{#> menu-body}}
+    {{#> menu-list}}
+      {{#> menu-list-item}}
+        {{#> menu-item menu-item--modifier="pf-m-selected"}}
+          {{#> menu-item-main}}
+            {{#> menu-item-text}}
+              Option 1
+            {{/menu-item-text}}
+            {{> menu-item-select-icon}}
+          {{/menu-item-main}}
+        {{/menu-item}}
+      {{/menu-list-item}}
+      {{#> menu-list-item}}
+        {{#> menu-item menu-item--modifier="pf-m-selected"}}
+          {{#> menu-item-main}}
+            {{#> menu-item-text}}
+              Option 2
+            {{/menu-item-text}}
+            {{> menu-item-select-icon}}
+          {{/menu-item-main}}
+        {{/menu-item}}
+      {{/menu-list-item}}
+      {{#> menu-list-item}}
+        {{#> menu-item menu-item--modifier="pf-m-selected"}}
+          {{#> menu-item-main}}
+            {{#> menu-item-icon}}
+              <i class="fas fa-fw fa-table" aria-hidden="true"></i>
+            {{/menu-item-icon}}
+            {{#> menu-item-text}}
+              Option 3
+            {{/menu-item-text}}
+            {{> menu-item-select-icon}}
+          {{/menu-item-main}}
+          {{#> menu-item-description}}
+            Description
+          {{/menu-item-description}}
+        {{/menu-item}}
+      {{/menu-list-item}}
+    {{/menu-list}}
+  {{/menu-body}}
+{{/menu}}
+```
+
+### Responsive-drill-down
```

Comment:
Do you want to leave this example or remove it until we support that?

Reply:
Yeah good point, i was following the designs in the marvel doc here: https://marvelapp.com/prototype/dhj94fe/screen/66701095 but ill wait til matt implements it completely